infra/resources/_modules/cosmos/cosmos_account.tf

```
@@ -0,0 +1,47 @@
+module "cosmos_account_wallet" {
+  source = "github.com/pagopa/terraform-azurerm-v3//cosmosdb_account?ref=v8.12.2"
+
+  name                = "${var.project}-cosmos-wallet"
```

```suggestion
  name                = "${var.project}-wallet-cosno-01"
```

[Source](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#databases)

---

done in https://github.com/pagopa/io-wallet/pull/56/commits/3f1ae7f83db0977b986e80e4ba6cbf4590b647b6